src/main/java/net/minecraftforge/fmp/microblock/IMicroblockContainerTile.java

```diff
@@ -0,0 +1,34 @@
+package net.minecraftforge.fmp.microblock;
+
+import net.minecraft.tileentity.TileEntity;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.fmp.multipart.IMultipartContainer;
+import net.minecraftforge.fmp.util.IWorldLocation;
+
+/**
+ * Interface that represents a {@link TileEntity} which can contain microblocks (but not other multiparts). You can add
+ * this to your {@link TileEntity} using {@link Capability Capabilities}.
+ *
+ * @see IMicroblock
+ * @see IMultipartContainer
```

Comment:
Yeeeeeeeeeahhhh... I may or may not have forgotten... It's already fixed, though :)